CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ The new default removes the linked file from the entry instead of deleting the f
 - The group editing window can now also be called by double-clicking the group to be edited. [koppor#277](https://github.com/koppor/jabref/issues/277)
 - The magnifier icon at the search shows the [search mode](https://help.jabref.org/en/Search#search-modes) again. [#3535](https://github.com/JabRef/jabref/issues/3535)
 - We added a new cleanup operation that replaces ligatures with their expanded form. [#3613](https://github.com/JabRef/jabref/issues/3613)
+- Pressing ESC while searching will clear the search field and select the first entry, if available, in the table. [koppor#293](https://github.com/koppor/jabref/issues/293)
```

Comment:
Just one minor thing: you should wrap keycodes in kbd tags, like `<kbd>ESC</kbd>`

Reply:
Oh so that's what was meant by wrapping keys in the changelog. Didn't know it actually means keycodes. My bad.

Reply:
While you are on it, could you do that here? <kbd>ESC</kbd>?